src/main/drivers/sensor.h

```diff
@@ -50,6 +50,8 @@ typedef bool (*sensorGyroInterruptStatusFuncPtr)(struct gyroDev_s *gyro);
 struct magDev_s;
 typedef bool (*sensorMagInitFuncPtr)(struct magDev_s *mag);
 typedef bool (*sensorMagReadFuncPtr)(struct magDev_s *mag);
+struct temperatureDev_s;
```

Comment:
I know you probably wrote this to respect the conventions in the file, but I think we should change this unspoken rule at some point (not necessarily right now). Instead of forward declaring `struct foo_s` and then declaring arguments as `struct foo_s` we could forward declare `foo_t` like `typedef struct foo_s foo_t` and then use `foo_t` for the argument types. This should work in the same cases as forward declaring the struct (its size is not required at compile time), while being a bit easier to read. What do you guys think?

Reply:
Yes I agree.